setup.py

```diff
@@ -51,10 +51,9 @@ def get_version():
         "opencv-python",
         "huggingface_hub",
         "pycocotools",
-        "wget"
-        # Stanford MIMI fork of TotalSegmentor
-        # FIXME: Figure out how to add git-based dependendcies.
-        # "git+https://github.com/StanfordMIMI/TotalSegmentator.git",
+        "wget",
+        "tensorflow",
```

Comment:
```suggestion
        "tensorflow>=2.0.0",
```

Reply:
We should mention that tensorflow should be installed before this library if users are looking for GPU support. TF install guides can be found here: https://www.tensorflow.org/install/pip